bedrock/src/main/java/no/apps/bedrock/utils/extensions/NetworkExtenstions.kt

```diff
@@ -0,0 +1,16 @@
+package no.apps.bedrock.utils.extensions
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.net.ConnectivityManager
+import android.net.NetworkCapabilities
+
+val Context.connectivityManager: ConnectivityManager
+    get() = getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
+
+val Context.isOnline: Boolean
+    @SuppressLint("MissingPermission")
+    get() = connectivityManager.getNetworkCapabilities(connectivityManager.activeNetwork)?.run {
+        return@run hasCapability(NetworkCapabilities.NET_CAPABILITY_INTERNET)
```

Comment:
You may use `let` instead of `run` so you don't have to explicitly return a value (last line of `let` is the return value of the block)